go.mod

```diff
@@ -6,20 +6,20 @@ require (
 	github.com/ghodss/yaml v1.0.0
 	github.com/go-bindata/go-bindata v3.1.2+incompatible
 	github.com/gonum/graph v0.0.0-20190426092945-678096d81a4b
-	github.com/openshift/api v0.0.0-20240405095747-3e50337fcaea
+	github.com/openshift/api v0.0.0-20240814144229-3f284e60eb8c
 	github.com/openshift/build-machinery-go v0.0.0-20231128094528-1e9b1b0595c8
 	github.com/openshift/client-go v0.0.0-20240405120947-c67c8325cdd8
 	github.com/openshift/library-go v0.0.0-20240412151150-149aa7608754
 	github.com/spf13/cobra v1.7.0
 	github.com/stretchr/testify v1.8.4
 	go.etcd.io/etcd/client/v3 v3.5.10
-	k8s.io/api v0.29.0
+	k8s.io/api v0.30.1
 	k8s.io/apiextensions-apiserver v0.29.0
-	k8s.io/apimachinery v0.29.0
+	k8s.io/apimachinery v0.30.1
 	k8s.io/apiserver v0.29.0
```

Comment:
Please also bump other k8s components to make sure they are all on 1.30.1 version.

Reply:
done. had to update an import here: `pkg/operator/revisionpoddeployer/deployers_test.go`